healthvault/src/androidTest/java/com/microsoft/healthvault/HVSettings.java

```diff
@@ -46,7 +46,7 @@ public void setShellUrl(String url) {
 	@Override
 	public String getRestUrl() {
 		// TODO Auto-generated method stub
-		return null;
+		return "https://data.ppe.microsofthealth.net";
```

Comment:
Some kind of config or at least constants file to reference things like urls will be cleaner